### PR TITLE
add docs for dove.toml manifest file

### PR DESCRIPTION
**Pull request by mkurnikov**

### CODE REVIEW

File: toolset/dove_toml_manifest.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+It consists of the following sections: 
+
+* [`[package]`](#the-package-section) 
+    * `name` - The name of the project
+      
+    * [`account_address`](#the-account_address-field) - The user account address
+
+    * [`dialect`](#the-dialect-field) - Address format of the project 
+      
+    * [`dependencies`](#the-dependencies-field) - Specifies sets of project dependencies
```
Comment (vitvakatu) [inline toolset/dove_toml_manifest.md:14]:
```suggestion
    * [`dependencies`](#the-dependencies-field) - Specifies a set of project dependencies
```
or 
```suggestion
    * [`dependencies`](#the-dependencies-field) - Specifies project dependencies
```

File: toolset/dove_toml_manifest.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+    { git = "https://github.com/pontem-network/move-stdlib", branch = "v1.2.0" },
+    { path = "./local_modules" }
+]
+```
+
+### The `account_address` field
+
+The address of the user account. 
+
+1. You can add `{{sender}}` clause to your source code,
```
Comment (vitvakatu) [inline toolset/dove_toml_manifest.md:38]:
```suggestion
1. You can use `{{sender}}` clause anywhere in your source code, 
```

File: toolset/dove_toml_manifest.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+### The `account_address` field
+
+The address of the user account. 
+
+1. You can add `{{sender}}` clause to your source code, 
+   which will be replaced with the value of `account_address` at compilation time.
+    
+2. If you `dove run` the script without `--signers` flag, 
+   this address will be a default signer passed to script function.
```
Comment (vitvakatu) [inline toolset/dove_toml_manifest.md:42]:
```suggestion
   this address will be the default `signer` passed to the script function.  
```

File: toolset/dove_toml_manifest.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+
+The address of the user account. 
+
+1. You can add `{{sender}}` clause to your source code, 
+   which will be replaced with the value of `account_address` at compilation time.
+    
+2. If you `dove run` the script without `--signers` flag, 
+   this address will be a default signer passed to script function.  
+
+3. If you don't wrap your module in the `address {}` block,
```
Comment (vitvakatu) [inline toolset/dove_toml_manifest.md:44]:
```suggestion
3. If you don't wrap your module with the `address {}` block, 
```

File: toolset/dove_toml_manifest.md [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+The address of the user account. 
+
+1. You can add `{{sender}}` clause to your source code, 
+   which will be replaced with the value of `account_address` at compilation time.
+    
+2. If you `dove run` the script without `--signers` flag, 
+   this address will be a default signer passed to script function.  
+
+3. If you don't wrap your module in the `address {}` block, 
+   this is the address where that module will be compiled at.
```
Comment (vitvakatu) [inline toolset/dove_toml_manifest.md:45]:
```suggestion
   this value is used as the address of the module. 
```

**Review by vitvakatu — COMMENTED** [review]
I'm not a native but some phrases look not natural to me, left some comments.

**Review by borispovod — APPROVED** [review]

**Pull request merged**